src/main/java/PersonalBudget/business/expense/domain/service/ExpenseTemplateService.java

```diff
@@ -0,0 +1,28 @@
+package PersonalBudget.business.expense.domain.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.ui.Model;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@Service
+public class ExpenseTemplateService {
+
+    private final ExpenseService expenseService;
+
+    public void addExpenseCategoriesAttribute(Model model) {
+        List<String> expenseCategories = expenseService.getExpenseCategoriesAssignedToUser();
```

Comment:
There can be a lot of "AssignedTo" if we have a bigger application. I recommend to stick to simple english method names such as getUserExpenseCategories -> I believe it means the same but it is smoother somehow :)